packages/vue/src/DataDisplay/Overview/OcOverviewItem.vue

```diff
@@ -82,7 +82,7 @@ defineProps({
               <slot name="content">
                 <div
                   class="flex items-center gap-x-3 overflow-hidden"
-                  v-if="percentValue"
+                  v-if="percentValue !== undefined"
```

Comment:
```suggestion
                  v-if="!isNaN(percentValue)"
```
It's better to test undefined, null, not number values, boolean